--- diskdev_cmds-540.1.linux3/newfs_hfs.tproj/makehfs.c~	2012-02-02 23:36:59.000000000 +0100
+++ diskdev_cmds-540.1.linux3/newfs_hfs.tproj/makehfs.c	2018-10-23 00:01:04.652419886 +0200
@@ -38,7 +38,9 @@
 #endif
 #include <sys/errno.h>
 #include <sys/stat.h>
+#ifndef __ILP32__
 #include <sys/sysctl.h>
+#endif
 #if !LINUX
 #include <sys/vmmeter.h>
 #endif
--- diskdev_cmds-540.1.linux3/fsck_hfs.tproj/fsck_hfs.c~	2012-02-16 20:59:54.000000000 +0100
+++ diskdev_cmds-540.1.linux3/fsck_hfs.tproj/fsck_hfs.c	2018-10-23 00:01:35.625938707 +0200
@@ -30,7 +30,9 @@
 #include <sys/ioctl.h>
 #if !LINUX
 #include <sys/disk.h>
+#ifndef __ILP32__
 #include <sys/sysctl.h>
+#endif
 #include <setjmp.h>
 #endif
 #include <hfs/hfs_mount.h>
--- diskdev_cmds-540.1.linux3/fsck_hfs.tproj/utilities.c~	2012-02-01 18:17:19.000000000 +0100
+++ diskdev_cmds-540.1.linux3/fsck_hfs.tproj/utilities.c	2018-10-23 00:02:03.069484085 +0200
@@ -70,7 +70,9 @@
 #include <string.h>
 #include <unistd.h> 
 #include <stdlib.h>
+#ifndef __ILP32__
 #include <sys/sysctl.h>
+#endif
 
 #include "fsck_hfs.h"
 
--- diskdev_cmds-540.1.linux3/fsck_hfs.tproj/dfalib/Scavenger.h~	2012-02-01 18:17:19.000000000 +0100
+++ diskdev_cmds-540.1.linux3/fsck_hfs.tproj/dfalib/Scavenger.h	2018-10-23 00:02:23.689671397 +0200
@@ -48,7 +48,9 @@
 #endif
 #include <sys/errno.h>
 #include <sys/param.h>
+#ifndef __ILP32__
 #include <sys/sysctl.h>
+#endif
 #include <sys/mount.h>
 #include <hfs/hfs_mount.h>
 
